src/main/java/com/codecool/barbershop/barbershop/client/ClientController.java

```diff
@@ -20,7 +21,8 @@ public ClientController(ClientService clientService) {
 
     @GetMapping
     public List<Client> getAllClients() {
-        return clientService.getAllClients();
+        Sort sort = Sort.by("firstName");
```

Comment:
Why not allow for some flexibility and allow a parameter that would indicate what do you want to sort by.  
Of course the unimplemented sorts would not be available, but you wouldn't have to hard code the column names here.